### ISSUE TITLE
[Bug Report] v-dialog lazy works only once

### ISSUE THREAD

**Issue by mosinve** [vuetifyjs/vuetify#5768]
### Versions and Environment
**Vuetify:** 1.3.11
**Vue:** 2.5.17
**Browsers:** Chrome 70.0.3538.102
**OS:** Mac OS 10.14.1

### Steps to reproduce
1. Open reproduction link example
2. Open browser devtool
3. Find `.v-dialog__content` element
4. Check that markup doesn't have dialog contents: 
```html
<div tabindex="-1" class="v-dialog__content" style="z-index: 0;"><div class="v-dialog" style="width: 500px; display: none;"></div></div>
```
5. Open && Close dialog.
6. Repeat steps 2-3.
7. Check that even dialog closed, now it's contents are in DOM:
```html
<div tabindex="-1" class="v-dialog__content" style="z-index: 202;"><div class="v-dialog" style="width: 500px; display: none;"> <div class="v-card theme--light"><div class="v-card__title headline grey lighten-2 v-card__title--primary">Privacy Policy</div> <div class="v-card__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div> <hr class="v-divider theme--light"> <div class="v-card__actions"><div class="spacer"></div> <button type="button" class="v-btn v-btn--flat theme--light primary--text"><div class="v-btn__content">I accept</div></button></div></div></div></div>
```

### Expected Behavior
Steps 4 and 7 should give the same results

### Actual Behavior
Steps 4 and 7 result are different

### Reproduction Link
<a href="https://codepen.io/cih/full/pQQdmz/" target="_blank">https://codepen.io/cih/full/pQQdmz/</a>

### Other comments
It seems, that current lazy handling linked to isBooted prop it incorrect, so `isBooted===false` only once, it would be better to use isActive instead of isBooted

**Comment by KaelWD** [comment]
Duplicate of #4502

**Comment by mosinve** [comment]
How this issue relates to #4502?? This is about v-dialog bug, and that is about timepicker

**Comment by KaelWD** [comment]
#4502 is a subset of this, but the fix is the same.
